go/vt/wrangler/testlib/backup_test.go

```diff
@@ -59,23 +58,20 @@ func TestBackupRestore(t *testing.T) {
 	require.NoError(t, err)
 }
 
-// TODO: @rameez. I was expecting this test to fail but it turns out
-// we infer decompressor through compression engine in builtinEngine.
+// Given we infer decompressor through Manifest file.
 // It is only in xtrabackup where we infer decompressor through extension & BuiltinDecompressor param.
```

Comment:
This comment is no longer valid, correct?

Reply:
I just uncover couple of more cases for example if someone tries to move from external to buit-in and vice versa. I am sending an update ..